SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -3230,6 +3246,24 @@ void USpatialNetDriver::TryFinishStartup()
 					USpatialNetDriverDebugContext::EnableDebugSpatialGDK(DebugActorSubView, this);
 				}
 #endif
+
+#if WITH_GAMEPLAY_DEBUGGER
+				const FFilterPredicate GameplayDebuggerCompFilter = [this](const Worker_EntityId EntityId,
```

Comment:
I feel the filters are not necessary here.
You can do a sub view just on the component you added for the debugger actor.
It was done otherwise for the DebugInterface because we dynamically added the component, and that had obscure implication for view completeness.

Reply:
I tried this initially but it did not work, and I didn't investigate why - I'll take another look shortly.

Reply:
As discussed offline, this won't work so I'll stick to the present solution.